app/gui/src/project-view/components/widgets/DropdownWidget.vue

```diff
@@ -109,12 +130,17 @@ export interface DropdownEntry {
   position: relative;
   user-select: none;
   min-width: 100%;
-  margin-top: calc(0px - var(--dropdown-extend));
-  padding-top: var(--dropdown-extend);
   background-color: var(--dropdown-bg);
   border-radius: calc(var(--item-height) / 2 + var(--dropdown-padding));
   color: var(--dropdown-fg);
+}
 
+/** Optional class that extends the dropdown upwards, so that it nicely merges with the node’s port.
+ * Normally, only dropdowns that directly attached to a port are extended. 
+ */
```

Comment:
Formatted as docstring, so keep docstring rules:
```suggestion
/**
 * Optional class that extends the dropdown upwards, so that it nicely merges with the node’s port.
 * Normally, only dropdowns that directly attached to a port are extended. 
 */
```